content/en/_index.md

```diff
@@ -1068,5 +1320,9 @@ end
 
 [minitest]: https://github.com/minitest/minitest
 
+{{< /content-tab >}}
+{{< content-tab header="Terraform" >}}
+For information about testing, see the Terraform samples [contributing guide]
+(https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md).
```

Comment:
blocker: We don't link to separate CONTRIBUTING guides from the style guide. However, after this PR is merged, the terraform-docs-samples repo should remove all redundant content and link here.

If there are some universal considerations on terraform sample testing that should existing beyond that repository, then we can add them here. Also, there is a testing section further up in the style guide that only lists Coverage currently, happy to discuss more guidelines there. We want to minimize the guidelines in the language-specific section at the bottom.

#StyleGuideFeedback

Reply:
Removed this and added two new sections to the general testing guidelines above. PTAL.